client/pages/ClubList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useContext } from "react";
+import ClubCard from "../components";
+import { AppContext } from "./_app";
+
+export default function ClubList() {
+  const appData = useContext(AppContext);
+  const clubListData = appData.clubs;
+
+  return (
+    <div>
+      <h1>Club List</h1>
+      {Array.from(clubListData.entries()).map(([clubId, clubInfo]) => {
+        return <ClubCard club={{ id: clubId, ...clubInfo }} />;
```

Comment:
Using the spread operator and other ES6 syntax might require babel/webpack to compile into CommonJS (so it will be supported by all browsers). Not an issue for modern browsers, but important for backwards compatibility.
See: https://css-tricks.com/transpiling-es6/#:~:text=While%20support%20for%20ES6%20is,need%20to%20transpile%20our%20code.

Reply:
(making another ticket for this to do at a later time)